questionnaires/serializers.py

```diff
@@ -0,0 +1,167 @@
+import json
+
+from rest_framework import serializers
+from wagtail.core.models import Page
+
+from questionnaires.models import (
+    Survey,
+    SurveyFormField,
+    Poll,
+    PollFormField,
+    QuizFormField,
+    Quiz,
+    UserSubmission,
+)
+
+
+class QuestionnairePageSerializer(serializers.ModelSerializer):
+    type = serializers.CharField(source='content_type.model')
+
+    class Meta:
+        model = Page
+        fields = ['id', 'title', 'type', 'last_published_at']
+
+
+class PollFormFieldSerializer(serializers.ModelSerializer):
+    choices = serializers.SerializerMethodField()
+    default_value = serializers.SerializerMethodField()
+
+    def get_choices(self, instance):
```

Comment:
Can we reduce the amount of duplication in this module? I suggest implementing one or two base classes and extending from those instead of `ModelSerializer` every time.

Reply:
done